api/caasapplication/client.go

```diff
@@ -64,14 +61,13 @@ type DeployArgs struct {
 // it. Placement directives, if provided, specify the machine on which the charm
 // is deployed.
 func (c *Client) Deploy(args DeployArgs) error {
-	deployArgs := params.ApplicationsDeploy{
-		Applications: []params.ApplicationDeploy{{
+	deployArgs := params.CAASApplicationsDeploy{
```

Comment:
Whoops :) I guess this worked anyway because the serialised structs were similar enough?